CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 kOS Mod Changelog
 =================
 
+# v0.19.3 In anticipation of 1.1
```

Comment:
I'd change the phrasing of this to "last planned update to KSP 1.0.5" just so nobody thinks this was meant to be a 1.1 compatible release.